src/app/view/components/listing-data-table/listing-data-table.component.html

```diff
@@ -1,47 +1,53 @@
 <dot-action-header [options]="actionHeaderOptions">
```

Comment:
This component is not working as expected, if you use it with 10 items per pege here: https://github.com/dotCMS/core-web/blob/issue-15455-Make-listing-data-table-component-navegable-with-keyboard/src/app/portlets/content-types/main/content-types.component.html#L7 is not showing the pagination correctly.